src/Sylius/Component/Core/Model/Shipment.php

```diff
@@ -45,4 +53,36 @@ public function setOrder(BaseOrderInterface $order = null)
 
         return $this;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStockLocation()
+    {
+        return $this->stockLocation;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setStockLocation(BaseStockLocationInterface $location = null)
+    {
+        $this->stockLocation = $location;
+
+        return $this;
```

Comment:
@pjedrzejewski @michalmarcinkowski What do you think about removing fluent interface in this case?